liquibase-core/src/main/java/liquibase/change/core/CreateProcedureChange.java

```diff
@@ -90,8 +84,7 @@ public void setEncoding(String encoding) {
     }
 
     @DatabaseChangeProperty(
-        description = "File containing the procedure text. Either this attribute or a nested procedure text is " +
-            "required.",
+        description = "File containing the procedure text. Either this attribute or a nested procedure text is required.",
```

Comment:
I'm reviewing this file from bottom to top so here's that word "nested" again. I'd say "inline" here, too. Or maybe, "Either this attribute or inline SQL within the `createProcedure` definition is required."